plugins/Time/plugin.py

```diff
@@ -206,7 +212,20 @@ def tztime(self, irc, msg, args, timezone):
         irc.reply(datetime.now(timezone).strftime(format))
     tztime = wrap(tztime, ['text'])
 
-
+    def ddate(self, irc, msg, args, year=None, month=None, day=None):
+        """[<year> <month> <day>]
+        Returns a the Discordian date today, or an optional different date."""
+        if _ddate is not None:
+            if year is not None and month is not None and day is not None:
+                try:
+                    irc.reply(_ddate(datetime(year=year, month=month, day=day)))
+                except ValueError as e:
+                    irc.error("%s", e)
+            else:
+                irc.reply(_ddate())
+        else:
+            irc.error("The 'ddate' module is not installed. Use 'pip install --user ddate' See <https://pypi.python.org/pypi/ddate/> for more information.")
```

Comment:
Could you split lines, wrap the string with `_()`, and use `"%s -m pip", sys.executable` instead of just `pip`?